Frends.FTP.ListFiles/Frends.FTP.ListFiles/Definitions/Result.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+
+namespace Frends.FTP.ListFiles.Definitions
+{
+    /// <summary>
+    /// List of files.
+    /// </summary>
+    public class Result
+    {
+        /// <summary>
+        /// List of files.
+        /// </summary>
+        public List<ListObject> Files { get; set; }
+
+        internal Result(List<ListObject> listObject)
```

Comment:
I do not mind really, but you could omit the constructor altogether and use the initializer syntax like `return new Result { Files = something };` Just as a suggestion :)